.oh/metis/reverify-plan-semantics-against-head.md

```
@@ -0,0 +1,28 @@
+---
+id: reverify-plan-semantics-against-head
+title: "Re-verify Plan-Stage Behavioural Notes Against HEAD Before Encoding Them"
+outcome: context-assembly
+source: PR #874 / issue #873 ship pipeline
+---
+An issue's root-cause analysis and a solution-space plan are snapshots of the
+code at the moment they were written. By the time `/execute` starts, sibling
+PRs may have landed and silently changed the semantics the plan promises to
+preserve. "Edge-for-edge identical" is then a claim about the wrong baseline.
+
+#873 was written against Pass 1 code that #800 had already half-replaced:
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)